### PR TITLE
Scaffold for homechef

### PR DESCRIPTION
**Pull request by kriizjones**
# Pivotal Tracker story

https://www.pivotaltracker.com/n/projects/1948009

# What I have learned working on this feature:

• Set up Ionic 

## Ready for review!

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "HelloIonic",
```
Comment (AmberWilkie) [inline bower.json]: Is this the name of your app?

File: www/js/app.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// Ionic Starter App
+
+// angular.module is a global place for creating, registering and retrieving Angular modules
+// 'starter' is the name of this angular module example (also set in a <body> attribute in index.html)
+// the 2nd parameter is an array of 'requires'
+// 'starter.controllers' is found in controllers.js
+angular.module('starter', ['ionic', 'starter.controllers'])
```
Comment (AmberWilkie) [inline www/js/app.js]: Is the name of your app 'starter'? That's what `starter` here means. You'll also have to rename your controllers: `homechef.controllers`, etc.

**Review by AmberWilkie — COMMENTED** [review]
Ok just a couple of name things - make sure you can still start up Ionic Labs and see the app before you push again.

File: bower.json (hunk 1 above)
Comment (kriizjones) [reply bower.json]: oh i miss that!

**Pull request merged**